### PR TITLE
Doc English doc for admin-api.md

### PR DESCRIPTION
**Pull request by juzhiyuan**
### Summary

English doc for [admin-api.md](https://github.com/apache/incubator-apisix/blob/feature-admin-api-doc/doc/admin-api.md)

### Full changelog

* [Added admin-api.md]

### Issues resolved

resolve https://github.com/apache/incubator-apisix/issues/935

### CODE REVIEW

File: doc/admin-api.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+|PUT      |/apisix/admin/routes/{id}|{...}|Create resource by ID|
+|POST     |/apisix/admin/routes     |{...}|Create resource, and ID is generated by server|
+|DELETE   |/apisix/admin/routes/{id}|NULL|Remove resource|
+|PATCH    |/apisix/admin/routes/{id}/{path}|{...}|Update targeted content|
+
+> URI Request Parameters：
+
+|parameter      |Required   |Type |Description        |Example|
+|---------|---------|----|-----------|----|
+|ttl     |False |辅助   |expired time at target seconds|ttl=1|
```
Comment (juzhiyuan) [inline doc/admin-api.md]: @moonming I'm not sure what does `辅助` means?

File: doc/admin-api.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+> Request Body Parameters：
+
+|Parameter      |Required   |Type |Description        |Example|
+|---------|---------|----|-----------|----|
+|desc     |False |辅助   |Identifies route names, usage scenarios, and more.|customer xxxx|
+|uri      |True |Match Rules|In addition to full matching such as `/foo/bar`、`/foo/gloo`, using different [Router](architecture-design-cn.md#router) allows more advanced matching, see [Router](architecture-design-cn.md#router) for more.|"/hello"|
+|host     |False |Match Rules|Currently requesting a domain name, such as `foo.com`; pan-domain names such as `*.foo.com` are also supported.|"foo.com"|
+|hosts    |False |Match Rules|The `host` in the form of a list means that multiple different hosts are allowed, and any one of them can be matched.|{"foo.com", "*.bar.com"}|
```
Comment (juzhiyuan) [inline doc/admin-api.md]: image

@moonming @membphis  I'm not sure if it's ok to translate `匹配其中任意一个即可` to `and any one of them can be matched`?

Comment (moonming) [reply doc/admin-api.md]: `match any one of them`?

File: doc/admin-api.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+|PUT      |/apisix/admin/routes/{id}|{...}|Create resource by ID|
+|POST     |/apisix/admin/routes     |{...}|Create resource, and ID is generated by server|
+|DELETE   |/apisix/admin/routes/{id}|NULL|Remove resource|
+|PATCH    |/apisix/admin/routes/{id}/{path}|{...}|Update targeted content|
+
+> URI Request Parameters：
+
+|parameter      |Required   |Type |Description        |Example|
+|---------|---------|----|-----------|----|
+|ttl     |False |辅助   |Expires after target seconds|ttl=1|
```
Comment (moonming) [inline doc/admin-api.md]: no translate `辅助`?

File: doc/admin-api.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+> Request Body Parameters：
+
+|Parameter      |Required   |Type |Description        |Example|
+|---------|---------|----|-----------|----|
+|desc     |False |辅助   |Identifies route names, usage scenarios, and more.|customer xxxx|
```
Comment (moonming) [inline doc/admin-api.md]: ditto

**Comment by juzhiyuan** [comment]
@moonming Could we merge this PR to master branch?

**Comment by moonming** [comment]
sure

Thanks,
Ming Wen
Twitter: _WenMing


琚致远 <notifications@github.com> 于2019年12月10日周二 上午8:25写道：

**Pull request merged**